react-app/src/app/screens/Game/layout.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Board from '../../components/Board';
+
+import './styles.css';
+
+export function ListItem({ desc, onClick }) {
+  return (
+    <li>
+      <button onClick={onClick}>{desc}</button>
+    </li>
```

Comment:
Why the `<li>` tags??

Reply:
Move listItem to the component folder

Reply:
If I remove it, the steps are not seen as a vertical list.

Reply:
How about a `<div/>` with column class?